pytorch/containers/published_container/README.md

```diff
@@ -0,0 +1,219 @@
+
+Citation: Dua, D. and Karra Taniskidou, E. (2017). UCI Machine Learning Repository
+[http://archive.ics.uci.edu/ml]. Irvine, CA: University of California, School of Information and
+Computer Science.
+
+# How to use a published container to train a PyTorch model
+1. Create your model
+1. Test your trainer application locally and package it
+1. Upload the package to Cloud Storage
+1. Submit your training job
+
+# Prerequisites
+Before you jump in, let’s cover some of the different tools you’ll be using to get your container
+up and running on ML Engine.
+
+[Google Cloud Platform](https://cloud.google.com/) lets you build and host applications and
+websites, store data, and analyze data on Google's scalable infrastructure.
+
+[Cloud ML Engine](https://cloud.google.com/ml-engine/) is a managed service that enables you to
+easily build machine learning models that work on any type of data, of any size.
+
+[Cloud Container Registry](https://cloud.google.com/container-registry/) is a single place for
+your team to manage Docker images, perform vulnerability analysis, and decide who can access what
+with fine-grained access control.
+
+[Google Cloud Storage](https://cloud.google.com/storage/) (GCS) is a unified object storage for
+developers and enterprises, from live data serving to data analytics/ML to data archiving.
+
+[Cloud SDK](https://cloud.google.com/sdk/) is a command line tool which allows you to interact
+with Google Cloud products. In order to run this tutorial, make sure that Cloud SDK is
+[installed](https://cloud.google.com/sdk/downloads) in the same environment as your Jupyter kernel.
+
+[docker](https://www.docker.com/) is a containerization technology that allows developers to
+package their applications and dependencies easily so that they can be run anywhere.
+
+# Part 0: Setup
+* [Create a project on GCP](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+* [Create a Google Cloud Storage Bucket](https://cloud.google.com/storage/docs/quickstart-console)
+* [Enable Cloud Machine Learning Engine, Container Registry, and Compute Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component,containerregistry.googleapis.com)
+* [Install Cloud SDK](https://cloud.google.com/sdk/downloads)
+* [Install docker](https://docs.docker.com/install/)
+* [Install PyTorch](https://pytorch.org/get-started/locally/) [Optional: used if running locally]
+* [Install pandas](https://pandas.pydata.org/pandas-docs/stable/install.html) [Optional: used if running locally]
+
+These variables will be needed for the following steps.
+
+**Replace these variables:**
+```
+# PROJECT_ID: your project's id. Use the PROJECT_ID that matches your Google Cloud Platform project.
+export PROJECT_ID=YOUR_PROJECT_ID
+
+# BUCKET_ID: the bucket id you created above.
+export BUCKET_ID=BUCKET_ID
+```
+Additional variables:
+```
+# PUBLISHED_IMAGE_URI: the complete URI location for Cloud Container Registry
```

Comment:
perhaps somehow highlight this part a bit more, since this seems the only difference from https://github.com/GoogleCloudPlatform/cloudml-samples/pull/357?

Reply:
Well they also have to package their application differently.